docs/product/sentry-basics/concepts/tracing/trace-view.mdx

```diff
@@ -4,47 +4,111 @@ sidebar_order: 30
+
+If you have highlighted areas of interest in your trace and want an easy way to switch between them, like for example to perform comparison, simply pin the tabs. When a tab is pinned, it will be persisted througout the session. Focusing on a tab will take you right to it's entry in the list, so you will always know exactly where is, without having to do any manual scrolling.
+
+<div
+  style={{
+    height: "0px",
+    paddingBottom: "calc(66.66% + 41px)",
+    position: "relative",
+    width: "100%",
+  }}
+>
+  <iframe
+    src="https://demo.arcade.software/yP3PFytK0ckfetjJTe52?embed&show_copy_link=true"
+    title="Sentry"
+    frameborder="0"
+    loading="lazy"
+    webkitallowfullscreen
+    mozallowfullscreen
+    allowfullscreen
+    allow="clipboard-write"
+    style={{
+      colorScheme: "light",
+      height: "100%",
+      left: "0px",
+      position: "absolute",
+      top: "0px",
+      width: "100%",
+    }}
+    title="Trace view - Tabbing"
+  ></iframe>
+</div>
+
+Sometimes, duration gaps between entries can be very small, yet still very critical to your debugging effort, this is why the new view enables you to zoom into certain areas of the trace, either by double clicking on row elements, which zooms into the row you are scrolling to, or by using your mouse wheel.
+
+<div
+  style={{
+    height: "0px",
+    paddingBottom: "calc(66.66% + 41px)",
+    position: "relative",
+    width: "100%",
+  }}
+>
+  <iframe
+    src="https://demo.arcade.software/U8JbMjFJR1KRiP4G4j4E?embed&show_copy_link=true"
+    title="Trace Details — Performance — sentry — Sentry"
+    frameborder="0"
+    loading="lazy"
+    webkitallowfullscreen
+    mozallowfullscreen
+    allowfullscreen
+    allow="clipboard-write"
+    style={{
+      colorScheme: "light",
+      height: "100%",
+      left: "0px",
+      position: "absolute",
+      top: "0px",
+      width: "100%",
+    }}
+    title="Trace view - Zooming"
+  ></iframe>
+</div>
+
+The trace view has other features like search, which makes it easy for you to visualize matching spans and iterate over them by pressing the up or down arrow keys as well as tailoring the layout to your preferences. That said, this is not then end, and we are going to continue adding features that enable you to perform efficient investigation into whatever issues you are facing. If you have feedback or requests, please dont hesitate to reach out as we would love to hear how we can make your life easiers.
```

Comment:
```suggestion
The trace view has other features like search, which makes it easy to visualize matching spans that you can then iterate over using the up or down arrows. You can also tailor the layout to your preference. 

We're going to continue adding features that enable you to perform efficient investigation into whatever issues you might be facing. If you have feedback or requests, please share it here (ADD LINK). We'd love to hear how we can make your life easier and your users happier.
```
I think this is the perfect spot to ask for feedback. Would be good to add a link.